components/Variation/basic_usage.rst

```diff
@@ -36,21 +41,18 @@ These variants should be generated by the ``VariantGenerator`` from the options
 8. Red, Medium
 9. Red, Large
 
-
-**The Code**
-
 .. code-block:: php
 
     use Sylius\Component\Resource\Repository\RepositoryInterface;
     use Sylius\Component\Variation\Generator\VariantGenerator;
     use Sylius\Component\Variation\Model\VariableInterface;
     use Sylius\Component\Variation\SetBuilder\CartesianSetBuilder;
- 
+
     /** @var RepositoryInterface $variantRepository */
     $variantRepository = /*...*/
 
     $setBuilder = new CartesianSetBuilder();
-    
+
     /**
      * From the example above, this would be the T-Shirt product.
      *
```

Comment:
Istead of using comment maybe create an instance of class which implements VariableInterface.